### ISSUE TITLE
Style/SpaceAroundOperators reports warning if hash is used in paralllel assignment

### ISSUE THREAD

**Issue by ojab** [bbatsov/rubocop#2836]
In code with parallel assignment like:
```ruby
x = {}
x[:foo], _y = 1, 2
```
rubocop (0.37.2 (using Parser 2.3.0.5, running on ruby 2.3.0 x86_64-darwin15)) reports warnings:
```
Surrounding space missing for operator `[:foo]`. [Style/SpaceAroundOperators]
```
that shouldn't be reported.

**Comment by ojab** [comment]
…and looks like it's already fixed in 504fe5fa9a1e64fcacfb8d583e2839ee5f35fd2c
